.github/workflows/ci.yml

```
@@ -18,11 +18,9 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-22.04, ubuntu-20.04]
-        elixir_version: [1.12.3, 1.13.3, 1.14.1]
-        otp_version: [24, 25]
-        exclude:
-          - otp_version: 25
-            elixir_version: 1.12.3
+        elixir_version: [1.13, 1.14, 1.15]
```

```suggestion
        elixir_version: [1.13, 1.14, 1.15, 1.16]
```

Elixir 1.16 was released recently, so it should be added to this. (And it may be too soon to leave out 1.13)

https://github.com/elixir-lang/elixir/releases/tag/v1.16.0

---

https://github.com/ExHammer/hammer/pull/79